chemprop/cli/train.py

```diff
@@ -814,6 +814,14 @@ def train_model(args, train_loader, val_loader, test_loader, output_dir, scaler,
         patience = args.patience if args.patience is not None else args.epochs
         early_stopping = EarlyStopping("val_loss", patience=patience, mode=monitor_mode)
 
+        if args.task_type == "regression":
```

Comment:
This won't actually work because then the model predictions will be unscaled during training. But I needed to add these to the model before calling trainer.fit so that the loc and scale show up in the autosaved checkpoints. I'm confused why this wasn't an issue with the checkpoints before because when I regenerated the checkpoints I hadn't changed any code related to loc and scale. 

In any event there is an open PR to improve the scalers and this won't be an issue after that is merged.